modules/pubxBidAdapter.js

```diff
@@ -15,9 +15,12 @@ export const spec = {
     return validBidRequests.map(bidRequest => {
       const bidId = bidRequest.bidId;
       const params = bidRequest.params;
+      const pageUrl = location.href.replace(/\?.*$/, '');
```

Comment:
why not use site.page on the request object?

Reply:
@patmmccann 
Thank you for letting me know about the site object. I found what I needed in the request object with `ortb2.site.page` .
Made the changes and committed the code. b2943068cbd223e0d7588d202b78a3891a30d921